index.html

```diff
@@ -0,0 +1,140 @@
+            </div>
+          </article>
+          <a
+            class="project-tile"
+            href="https://github.com/Zeph180/microversTrials"
+            id="profile-link"
+            target=" _blank"
+          >
+            <img
+              src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png"
+              alt="dr Norman"
+              class="image"
+            />
+            <div class="description">
+              <p>Javascript Calculator</p>
+            </div>
+          </a>
+          <article class="project-tile">
+            <img
+              src="https://cdn.freecodecamp.org/testable-projects-fcc/images/map.jpg"
+              alt="dr Norman"
+              class="image"
+            />
+            <div class="description">
+              <p>Map Data Across The Globe</p>
+            </div>
+          </article>
+          <a
+            class="project-tile"
+            href="https://codepen.io/freeCodeCamp/full/wGqEga"
+            target="_blank"
+          >
+            <img
+              src="https://cdn.freecodecamp.org/testable-projects-fcc/images/wiki.png"
+              alt="dr Norman"
+              class="image"
+            />
+            <div class="description">
+              <p>Wikipaedia Veiwer</p>
+            </div>
+          </a>
+          <article class="project-tile">
+            <img
+              src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tic-tac-toe.png"
+              alt="dr Norman"
+              class="image"
+            />
+            <div class="description">
+              <p>Tic Tac Toe Game</p>
+            </div>
+          </article>
+        </div>
+      </section>
+    </main>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <script src="js/index.js"></script>
+
+    <section class="contact-section">
+      <div class="" contact-section-header>
+        <h2>Lets's work together...</h2>
+        <p>How do you take your coffee</p>
+      </div>
+      <div>
+        <a
+          class="btn contact-details"
+          href="https://facebook.com/freecodecamp"
+          target="_blank"
+          >Facebook</a
+        >
+        <a
+          class="btn contact-details"
+          href="https://github.com/freeCodeCamp"
+          id="profile-link"
+          target="_blank"
+          >Github</a
+        >
+        <a class="btn contact-details" href="https://teitter.com/freeCodeCamp"
+          >Twitter</a
+        >
+        <a class="btn contact-details" href="mailto:example@example.com"
+          >Send a mail</a
+        >
+        <a class="btn contact-details" href="tel:555-555-5555">Call me</a>
+      </div>
+    </section>
+    <footer>
+      <p>
+        **This is just fake portfolio. All the projects and contact details
+        given are not real
+      </p>
+      <p>Created for freeCodeCamp</p>
+    </footer>
+  </body>
+</html>
```

Comment:
Html semantic tags a good way to organize a html document. The include Header Main Section etc. Lets break our html docu into those